reana_server/rest/gitlab.py

```diff
@@ -167,12 +172,20 @@ def gitlab_oauth(user):  # noqa
                 "code": gitlab_code,
                 "grant_type": "authorization_code",
             }
-            gitlab_response = requests.post(
-                REANA_GITLAB_URL + "/oauth/token", data=params
-            ).content
+
+            # request access token
+            client = GitLabClient()
```

Comment:
Cosmetics: since we may have a REANA client in the module, it may be good to be more specific and instead of `client` use `gitlab_client` everywhere as the symbol name? Currently some lines have shorter name, some longer:

```
reana_server/rest/gitlab.py:177:            client = GitLabClient()
reana_server/rest/gitlab.py:182:            authenticated_client = GitLabClient(access_token=access_token)
reana_server/rest/gitlab.py:318:        client = GitLabClient.from_k8s_secret(user.id_)
reana_server/rest/gitlab.py:476:        client = GitLabClient.from_k8s_secret(user.id_)
reana_server/utils.py:447:    client = GitLabClient.from_k8s_secret(user_id)
reana_server/utils.py:467:        client = GitLabClient.from_k8s_secret(user.id_)
tests/test_gitlab_client.py:40:     gitlab_client = GitLabClient.from_k8s_secret(user_id, host="gitlab.example.org")
tests/test_gitlab_client.py:70:     gitlab_client = GitLabClient(
tests/test_gitlab_client.py:92:     gitlab_client = GitLabClient(
tests/test_gitlab_client.py:113:    gitlab_client = GitLabClient(
tests/test_gitlab_client.py:133:    gitlab_client = GitLabClient(
tests/test_gitlab_client.py:166:    gitlab_client = GitLabClient(
tests/test_gitlab_client.py:188:    gitlab_client = GitLabClient(
tests/test_gitlab_client.py:210:    gitlab_client = GitLabClient(
tests/test_gitlab_client.py:235:    gitlab_client = GitLabClient(
tests/test_gitlab_client.py:261:    gitlab_client = GitLabClient(
```

Just cosmetics, since the context is usually very clear...